tests/test_util.py

```diff
@@ -0,0 +1,196 @@
+from unittest import TestCase
```

Comment:
`test_util.py` made me think that this file contains utils for the tests. Could you change the filename to something like `test_ain_utils.py`?

Reply:
Also, it would be very helpful if there was a short readme on how to run the test!

Reply:
Done!